deployment/files/traefik/deployment.yml

```diff
@@ -28,10 +28,9 @@ spec:
               cpu: "250m"
             limits:
               memory: "200Mi"
-              cpu: "500m"
+              cpu: "2"
```

Comment:
noticed that Traefik was hitting the limit, so increased it to use a full node. CPU usage is high during TLS termination.